cloud-providers/azure/kubernetes/payara-micro-distributed-cache/src/test/java/be/rubus/accelerator/CallEndpoint.java

```diff
@@ -0,0 +1,51 @@
+package be.rubus.accelerator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.SocketException;
+import java.net.URL;
+import java.util.Date;
+
+public class CallEndpoint {
```

Comment:
This test program is not needed in this case since we didn't configure liveness and readiness so we can't test out the scaling.